packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -20,22 +20,6 @@ import BlockIcon from '../block-icon';
 import { InserterListboxItem } from '../inserter-listbox';
 import InserterDraggableBlocks from '../inserter-draggable-blocks';
 
-/**
- * Return true if platform is MacOS.
- *
- * @param {Object} _window window object by default; used for DI testing.
- *
- * @return {boolean} True if MacOS; false otherwise.
- */
-function isAppleOS( _window = window ) {
```

Comment:
@talldan I reused the export form the key codes package for these two other instances. As you can see we're always making this check in the context of key codes, so I think it makes sense in this package.